unique_sdk/unique_sdk/_http_client.py

```diff
@@ -56,6 +59,7 @@ def _request_internal(self, method, url, headers, post_data, is_streaming):
                     data=json.dumps(post_data),
                     timeout=self._timeout,
                     **kwargs,
```

Comment:
Flip verify and **kwargs

```suggestion
                    verify=verify,
                    **kwargs,
```